sdk/core/Azure.Core/src/Pipeline/HttpClientTransport.Response.cs

```diff
@@ -42,7 +42,7 @@ internal void SetContentStream(Stream stream)
             {
                 ClientUtilities.AssertNotNull(stream, nameof(stream));
 
-                Content = MessageBody.CreateContent(stream);
+                Content = MessageBody.CreateBody(stream);
```

Comment:
Why is the method not called simply Create? It's a static method on MessageBody, so it's already implied that it creates the body.